### DISCUSSION TITLE
Installed the Mod

### DISCUSSION THREAD

**Discussion by Lino555** [Arkensor/EnfusionPersistenceFramework#33] [Q&A]
Hello,

I installed the mod and I see its working creating the save files for the playersave, sessions, and settings. But when I reboot the server it does not load those saves.

**Comment by Arkensor** [comment]
Hello,
I need way more information than this to say anything. Logs? Db directory? Is your game mode even setup for this?

**Comment by kOepi123** [comment]
@Lino555 I believe you are using the vanilla system and monitoring that one. The vanilla system is still broken to my own rigorous testing.
